app/src/jira-client/delete-builds.ts

```diff
@@ -16,11 +17,9 @@ async function deleteBuilds(cloudId: string, jenkinsServerUuid?: string): Promis
 		url += `&jenkinsServerUuid=${jenkinsServerUuid}`;
 	}
 
-	// @ts-ignore // required so that Typescript doesn't complain about the missing 'api' property
-	// eslint-disable-next-line no-underscore-dangle
-	const apiResponse = await global.api
+	const apiResponse = await api
 		.asApp()
-		.__requestAtlassian(url, {
+		.requestJira(route`${url}`, {
```

Comment:
This is the main change across the 4 calls in this PR,
uses requestJira now instead and requires a Route object